Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -282,6 +302,9 @@ void TextEditor::doubleclick_event(MouseEvent& event)
         m_selection.set_end(document().first_word_break_after(position));
     }
 
+    // FIXME: This does not reset the spans attributes after selection is completed thus all the occurances are still
+    //        highlighted even if we click on the editor after selection.
```

Comment:
Overriding `TextEditor::click_event()` and calling `document().set_spans(whatever_index, {});` would clear this nicely.

Reply:
ahh thats much better solution. :)